serveradmin/apps/management/commands/createapp.py

```diff
@@ -0,0 +1,60 @@
+"""Serveradmin
+
+Create application for user
+
+Copyright (c) 2021 InnoGames GmbH
+"""
+from os import environ
+
+from django.contrib.auth.models import User
+from django.core.management import BaseCommand, CommandError
+
+from serveradmin.apps.models import Application
+
+
+class Command(BaseCommand):
+    help = 'Create application for user'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--owner', help='Username of application owner')
+        parser.add_argument('--superuser', action='store_true',
+                            help='Create superuser token')
+        parser.add_argument('--non-interactive', action='store_true', help=(
+            'Gather values from SERVERADMIN_TOKEN_OWNER, SERVERADMIN_TOKEN '
+            'and SERVERADMIN_TOKEN_SUPERUSER env variables. '
+            'Any value for SERVERADMIN_TOKEN_SUPERUSER means yes. '
+            'Empty string or absence means no.'
+        ))
+
+    def handle(self, *args, **options):
+        if options['non_interactive']:
+            owner = environ.get('SERVERADMIN_TOKEN_OWNER')
+            token = environ.get('SERVERADMIN_TOKEN', default='')
+            superuser = bool(
+                environ.get('SERVERADMIN_TOKEN_SUPERUSER', default=False))
+        else:
+            if options['owner']:
+                owner = options['owner']
+            else:
+                owner = input('Username of application owner (must exist!): ')
+            token = input('Token (empty for secure auto generated one): ')
+            superuser = options['superuser']
+
+        user = User.objects.filter(username=owner)
+        if not user.exists():
+            raise CommandError(f'No such user {owner} found!')
+
+        app = Application.objects.filter(
+            owner=user.get(), name=f'default app for {owner}')
```

Comment:
In general (might be out of scope): If this is our indication of a default app (which we should also change in general ^^), then it would be good to have a central point of getting that specific name, so typos and change of the string is no problem in the future.

Reply:
Not really. This is just for creating the development user and having a common string we can match to find the "default" application that should get created for the development environment.

Reply:
So, this name is just the name of the scope of this exact command here? Then nevermind. A const might be interesting though to not duplicate the string in line 48 and 55, but totally minor.